packages/jaeger-ui-components/src/settings/SpanBarSettings.tsx

```diff
@@ -0,0 +1,114 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import {
+  DataSourceJsonData,
+  DataSourcePluginOptionsEditorProps,
+  GrafanaTheme,
+  toOption,
+  updateDatasourcePluginJsonDataOption,
+} from '@grafana/data';
+import { InlineField, InlineFieldRow, Input, Select, useStyles } from '@grafana/ui';
+
+export interface SpanBarOptions {
+  type?: string;
+  tag?: string;
+  process?: string;
+}
+
+export interface SpanBarOptionsData extends DataSourceJsonData {
+  spanBar?: SpanBarOptions;
+}
+
+export const NONE = 'None';
+export const DURATION = 'Duration';
+export const TAG = 'Tag';
+export const PROCESS = 'Process';
+
+interface Props extends DataSourcePluginOptionsEditorProps<SpanBarOptionsData> {}
+
+export default function SpanBarSettings({ options, onOptionsChange }: Props) {
+  const styles = useStyles(getStyles);
+  const selectOptions = [NONE, DURATION, TAG, PROCESS].map(toOption);
```

Comment:
Wdyt about combining tag and process as a single option and labeling it tag? For example, in trace to logs and trace to metrics, we don't separate them in the configuration. When constructing the query, we join the process and span tags to search for the configured value. Differentiating span vs. process tags might be confusing to some users.